### PR TITLE
feat: enable remote stake delegation

### PR DESCRIPTION
**Pull request by guibescos**
I want to add this to be able to delegated Pythnet PGAS stake to new validators.

### CODE REVIEW

File: governance/xc_admin/packages/xc_admin_cli/src/index.ts [hunk 1]
```diff
@@ -178,7 +187,11 @@ multisigCommand(
     })
     .instruction();
 
-  await vault.proposeInstructions([proposalInstruction], targetCluster);
+  await vault.proposeInstructions(
+    [proposalInstruction],
+    targetCluster,
+    DEFAULT_PRIORITY_FEE_CONFIG
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_cli/src/index.ts:193]: drive-by add priority fees

File: governance/xc_admin/packages/xc_admin_cli/src/index.ts [hunk 2]
```diff
@@ -74,7 +75,7 @@ async function loadVaultFromOptions(options: any): Promise<MultisigVault> {
   const vault: PublicKey = new PublicKey(options.vault);
 
   const squad = SquadsMesh.endpoint(
-    getPythClusterApiUrl(multisigCluster),
+    options.rpcUrlOverride || getPythClusterApiUrl(multisigCluster),
```
Comment (keyvankhademi) [inline governance/xc_admin/packages/xc_admin_cli/src/index.ts:78]: It's generally better to use `??` for these conditionals.

File: governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/SolanaStakingMultisigInstruction.ts [hunk 3]
```diff
@@ -54,6 +55,31 @@ export class SolanaStakingMultisigInstruction implements MultisigInstruction {
         case "Authorize":
         case "AuthorizeWithSeed":
         case "Delegate":
+          const decodedDelegate = StakeInstruction.decodeDelegate(instruction);
+          return new SolanaStakingMultisigInstruction(
+            "Delegate",
+            {},
```
Comment (keyvankhademi) [inline governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/SolanaStakingMultisigInstruction.ts:61]: Now Authorize and AuthorizeWithSeed with also run the "Delegate" code since there are no breaks. Also "Deactivate" will also run the Delegate code. I think this was unintentional. We should move the Delegate above the Authorize and add break for both "Deactivate" and "Deligate"

Comment (guibescos) [reply governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/SolanaStakingMultisigInstruction.ts]: Great catch. Not adding `break` because it returns inside the case block

File: governance/xc_admin/packages/xc_admin_cli/src/index.ts (hunk 2 above)
Comment (guibescos) [reply governance/xc_admin/packages/xc_admin_cli/src/index.ts]: Yes

File: governance/xc_admin/packages/xc_admin_cli/src/index.ts [hunk 4]
```diff
@@ -315,7 +340,40 @@ multisigCommand(
...
+    );
+  });
+
+multisigCommand(
+  "delegate-stake",
+  "Delegate a stake account to the given vote account"
+)
+  .requiredOption("-s, --stake-account <pubkey>", "stake account to delegate")
+  .requiredOption("-d, --vote-account <pubkey>", "vote account to delegate to")
+  .action(async (options: any) => {
```
Comment (keyvankhademi) [inline governance/xc_admin/packages/xc_admin_cli/src/index.ts:356]: Is there anyway we can make this `option` typed?
